python/nav/mibs/alcatel_ind1_port_mib.py

```diff
@@ -0,0 +1,74 @@
+"""A class for getting DDM values for ALE equipment
+"""
+
+from twisted.internet import defer
+from twisted.internet.defer import returnValue
+from nav.mibs.mibretriever import MibRetriever
+from nav.models.manage import Sensor
+
+COLUMNS = {
+    "ddmPortTemperature": {
+        "unit_of_measurement": Sensor.UNIT_CELSIUS,
+        "precision": 3,
+        "scale": Sensor.SCALE_MILLI,
+        "name": "{ifc} Module Temperature",
+        "description": "{ifc} Module Temperature",
+    },
+    "ddmPortTxBiasCurrent": {
+        "unit_of_measurement": Sensor.UNIT_AMPERES,
+        "precision": 3,
+        "scale": Sensor.SCALE_MILLI,
+        "name": "{ifc} TX Laser Bias Current",
+        "description": "{ifc} TX Laser Bias Current",
+    },
+    "ddmPortTxOutputPower": {
+        "unit_of_measurement": Sensor.UNIT_DBM,
+        "precision": 3,
+        "scale": Sensor.SCALE_MILLI,
+        "name": "{ifc} TX Laser Output Power",
+        "description": "{ifc} TX Laser Output Power",
+    },
+    "ddmPortRxOpticalPower": {
+        "unit_of_measurement": Sensor.UNIT_DBM,
+        "precision": 3,
+        "scale": Sensor.SCALE_MILLI,
+        "name": "{ifc} RX Laser Input Power",
+        "description": "{ifc} RX Laser Input Power",
+    }
+}
+
+
+class AlcatelInd1PortMib(MibRetriever):
+    """MibRetriever for Alcatel Port Sensors"""
+    from nav.smidumps.alcatel_ind1_port_mib import MIB as mib
```

Comment:
Because of the changes introduced in #1924, this line needs to change for this PR to pass tests again.

In your import section, you should add 

```python
from nav.smidumps import get_mib
```

And then change this line to

```python
mib = get_mib('ALCATEL-IND1-PORT-MIB')
```